pwiz_tools/Skyline/Properties/Resources.Designer.cs

```diff
@@ -3520,6 +3520,51 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Collision Cross Section.
+        /// </summary>
+        public static string ChromatogramContextMenu_Collision_Cross_Section {
+            get {
+                return ResourceManager.GetString("ChromatogramContextMenu_Collision_Cross_Section", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Compensation Voltage.
+        /// </summary>
+        public static string ChromatogramContextMenu_InsertIonMobilityMenuItems_Compensation_Voltage {

```

Comment:
Just not sure anyone is asking for this. It would be like showing the CE we used. I don't know of any instrument which does IMS filtering by CoV.

Reply:
Again. I see MS1 spectra from Thermo FAIMS DDA.